README.md

```diff
@@ -65,165 +67,6 @@ allprojects {
     }
 }
 ```
-
```

Comment:
I removed this section because we already have the usage documented in our official public documentation.